lib/iron_bank/describe/excluded_fields.rb

```diff
@@ -108,5 +165,6 @@ def extract_fields_from_exception(exception)
         failed_fields
       end
     end
+    # rubocop:enable Metrics/ClassLength
```

Comment:
extracting `##extract_fields_from_exception` and `#exctract_from_dividing` into service classes will make this a non-issue.